external/urlfrontier/pom.xml

```diff
@@ -34,6 +61,7 @@
 		<dependency>
 			<groupId>org.testcontainers</groupId>
 			<artifactId>testcontainers</artifactId>
+			<version>${testcontainer.version}</version>
```

Comment:
no need to specify the version here. the whole point of having it in the parent pom is tjhat it will be set there

Reply:
Resolved in https://github.com/DigitalPebble/storm-crawler/pull/982#discussion_r913721879